### PR TITLE
22.01 release notes

### PR DESCRIPTION
**Pull request by mvdbeek**
Please push or make suggestions as you see fit!

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: doc/source/releases/22.01_announce.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+**Galaxy starts as fastAPI application by default**
+  Starting galaxy via ``run.sh`` will use the new `gravity process manager
+  <https://github.com/galaxyproject/gravity>`__.
+  The new configuration uses `gunicorn <https://gunicorn.org/>`__ and `fastAPI <https://fastapi.tiangolo.com/>`__ to drive Galaxy's web process and starts job handler and celery processes.
+  For more details see the documentation at <WRITE AND PUT DOCS HERE>.
+  `Pull Request 13224`_
+
+**User Preferences can be encrypted in Galaxy Vault**
+  @nuwang, could you write something about this please?
```
Comment (mvdbeek) [inline doc/source/releases/22.01_announce.rst]: @nuwang do you have a moment to write a short summary here ?

File: doc/source/releases/22.01_announce_user.rst [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+* Add bioconductor as valid xref for tools
+  (thanks to `@bgruening <https://github.com/bgruening>`__).
+  `Pull Request 13268`_
+
+Release Testing Team
+===========================================================
+
+A special thanks to the release testing team for testing many of the new features and reporting many bugs:
+
+<team members go here>
```
Comment (mvdbeek) [inline doc/source/releases/22.01_announce_user.rst]: Should follow this format.
```suggestion
- `Keith Suderman <https://github.com/ksuderman>`_
```

Comment (Delphine-L) [inline doc/source/releases/22.01_announce_user.rst]:
```suggestion
- `Keith Suderman <https://github.com/ksuderman>`_
- `Delphine Lariviere <https://github.com/Delphine-L>`_
```

**Review by ksuderman — APPROVED** [review]

File: doc/source/releases/22.01_announce_user.rst [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+Release Testing Team
+===========================================================
+
+A special thanks to the release testing team for testing many of the new features and reporting many bugs:
+
+- `Keith Suderman <https://github.com/ksuderman>`_
+- `Delphine Lariviere <https://github.com/Delphine-L>`_
+- `Jennifer Hillman-Jackson <https://github.com/jennaj>`_
+
```
Comment (nakucher) [inline doc/source/releases/22.01_announce_user.rst]:
```suggestion
- `Natalie Kucher <https://github.com/nakucher>`_
- `Jennifer Hillman-Jackson <https://github.com/jennaj>`_
- `Alex Ostrovsky <https://github.com/astrovsky01>`_
- `Enis Afgan <https://github.com/afgane>`_

```

File: doc/source/releases/22.01_announce_user.rst [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  `Pull Request 13268`_
+
+Release Testing Team
+===========================================================
+
+A special thanks to the release testing team for testing many of the new features and reporting many bugs:
+
+- `Keith Suderman <https://github.com/ksuderman>`_
+- `Delphine Lariviere <https://github.com/Delphine-L>`_
+
```
Comment (ksuderman) [inline doc/source/releases/22.01_announce_user.rst:185]: + Alex Ostrovsky <https://github.com/astrovsky01?

Comment (ksuderman) [inline doc/source/releases/22.01_announce_user.rst]:
```suggestion
- `Jennifer Hillman-Jackson <https://github.com/jennaj>`_
- `Alex Ostrovsky <https://github.com/astrovsky01>`_
- `Enis Afgan <https://github.com/afgane>`_
 
```

File: doc/source/releases/22.01_announce.rst (hunk 1 above)
Comment (nuwang) [reply doc/source/releases/22.01_announce.rst]: Just saw this. Will do.

Comment (nuwang) [reply doc/source/releases/22.01_announce.rst]: Done.

Comment (mvdbeek) [reply doc/source/releases/22.01_announce.rst]: Thank you!

**Comment by mvdbeek** [comment]
Alright, I think the notes are ready for a final round of review & merging. Planning to release tomorrow.

File: doc/source/releases/22.01_announce.rst [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+Added
+-----
+
+The following configuration options are new
+
+config/galaxy.yml.sample:galaxy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  mapping.galaxy.mapping.enable_tool_source_display
```
Comment (nsoranzo) [inline doc/source/releases/22.01_announce.rst:71]: Do we want to keep all these `mapping.galaxy.mapping.` ?

**Pull request merged**